src/test/java/seedu/address/testutil/EditEventDescriptorBuilder.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.testutil;
+
+import seedu.address.logic.commands.event.EeditCommand;
+import seedu.address.model.module.Name;
+import seedu.address.model.module.event.Event;
+import seedu.address.model.module.event.EventDate;
+
+/**
+ * A utility class to help with building EditEventDescriptor objects.
+ */
+public class EditEventDescriptorBuilder {
+
+    private EeditCommand.EditEventDescriptor descriptor;
```

Comment:
Can change to private EditEventDescriptor desciptor instead. Import static seedu.address.logic.commands.event.EeditCommand.EditEventDescriptor should work. Do for the rest of the similars one below in this file